### DISCUSSION TITLE
Replacing branch drawing symbols in clipboard

### DISCUSSION THREAD

**Discussion by mattprecious** [ghostty-org/ghostty#2737] [Feature Requests, Ideas] [answered]
I'm adding support for the new branch drawing symbols (#2613) in my TUI and one drawback I've found is that they do not copy and paste well into other applications. The underlying unicode symbol is copied when in almost all cases I would like a semantically equivalent drawing symbol to be copied.

Is it possible for Ghostty (with or without help from my TUI) to automatically replace these symbols with portable alternatives when copying to the clipboard?

**Comment by mitchellh** [comment]
That'd be completely possible. What alternatives are you thinking?

**Reply by mattprecious** [reply]
In my TUI for unsupported terminals I use [white circle](https://unicodeplus.com/U+25CB), [black circle](https://unicodeplus.com/U+25CF), and the 'light' [box drawing characters](https://en.wikipedia.org/wiki/Box-drawing_characters#Box_Drawing) (specifically [horizontal](https://unicodeplus.com/U+2500), [vertical](https://unicodeplus.com/U+2502), [up and horizontal](https://unicodeplus.com/U+2534), and [up and left](https://unicodeplus.com/U+2518)). vim-flog seems to use the same set: https://github.com/rbong/vim-flog/blob/74e52ba4de25eb5f03824d65087750636d795641/lua/flog/graph.lua#L36-L86.

**Comment by mitchellh** [comment, marked as answer]
Finally promoted to issue #8383
